tests/test_config/test_partial_config.py

```diff
@@ -0,0 +1,98 @@
+import json
+import tempfile
+
+import bluepysnap.circuit as test_module
+from bluepysnap.sonata_constants import Edge, Node
+
+from utils import TEST_DATA_DIR
+
+
+def test_partial_circuit_config_minimal():
+    config = {
+        "metadata": {"status": "partial"},
+        "networks": {
+            "nodes": [
+                {
+                    # missing `nodes_file` rises libsonata exception
+                    "nodes_file": str(TEST_DATA_DIR / "nodes.h5"),
+                    "populations": {
+                        "default": {},
+                    },
+                }
+            ],
+            "edges": [
+                {
+                    # missing `edges_file` rises libsonata exception
+                    "edges_file": str(TEST_DATA_DIR / "edges.h5"),
+                    "populations": {
+                        "default": {},
+                    },
+                }
+            ],
+        },
+    }
```

Comment:
I believe this is the smallest config that does not produce errors in libsonata

I have tested everything I found on `NodePopulation` and `EdgePopulation` and nothing fails and everything returns reasonable values